src/main/java/uk/gov/hmcts/dts/fact/config/SecurityConfiguration.java

```diff
@@ -14,7 +14,7 @@
     protected void configure(HttpSecurity http) throws Exception {
         http
             .authorizeRequests(a -> a
-                .antMatchers(HttpMethod.GET, "/courts/all").authenticated()
+                .antMatchers(HttpMethod.GET, "/admin/courts").authenticated()
```

Comment:
do you need to put this back?

Reply:
yes i do but now im thinking, dont i need to do this for all admin request. i.e. the get and put method